### PR TITLE
New method to update user data to be sent to Amplitude

### PR DESCRIPTION
**Pull request by jantocv**
## Goals

Some analytic events requires certain user data. That user data can not be (sometimes) get by Native.
This new method send that new user data to Native side to be used to track certain events.

## Description

Method details provided in the analytics documentation. New user values can be sent independently, for example, if fullName has to be sent, the rest of the possible values can be empty.

### CODE REVIEW

File: docs/analytics.md [hunk 1]
```diff
@@ -17,3 +17,25 @@ Sends events to the different analytics provider configured in the native app.
...
+```
+Updates user information to be tracked on Analytic events.
+
+| Field | Type | Description |
+| ----- | ---- | ----------- |
+|  fullName | String | Name + Surname1 + Surname2  |
+|  country | String | Selected user country  |
+|  email | String | Email  |
+|  phoneNumber | String | Phone number  |
+|  gender | String | Gender  |
```
Comment (jasonadt) [inline docs/analytics.md:42]: Please add examples for the possible expected values for this line

**Review by jasonadt — CHANGES_REQUESTED** [review]
👍  looks great! Please address my minor comment.

**Comment by jasonadt** [comment]
@jantocv question: when should this new `udpateAnalyticsUserInfo` be called for a typical microApp/merino? Would it be just invoked once (whenever we have all the necessary data) or should we expect to have to invoke it each time before we send a new event? Please consider clarifying in the `analytics.md` file as well, thanks!

**Review by jasonadt — DISMISSED** [review]

**Comment by jantocv** [comment]
> @jantocv question: when should this new `udpateAnalyticsUserInfo` be called for a typical microApp/merino? Would it be just invoked once (whenever we have all the necessary data) or should we expect to have to invoke it each time before we send a new event? Please consider clarifying in the `analytics.md` file as well, thanks!

Updated! I didnt add anything regarding `merino` because it can be used form every repo that consumes AriesSDK.

**Review by rahulpsingh — APPROVED** [review]

**Pull request merged**